index.html

```diff
@@ -0,0 +1,164 @@
+            <section class="section3">
+                 <p>Guest Speakers and host</p>
+                 <hr class="newhr">
+                 <div class="speaker-container">
+                 <div class="speakers">
+                     <img class="images" src="images/Benimg.jpeg" alt="speakers imge">
+                     <div class="column">
+                     <h5>Ben Ghana</h5>
+                     <h4>Founder of the SSAMB </h4>
+                     <hr>
+                     <p>Ben Ghana founder of the SSAMB community. Putting all efforts to see girls are well oriented about tech. Tech is a master piece for ladies a well valued craft</p>
+                     </div>
+                     </div>
+                 <div class="speakers">
+                     <img class="images" src="images/Benimg.jpeg" alt="speakers imge">
+                     <div class="column">
+                     <h5>Mike Nigeria</h5>
+                     <h4>Co-founder of the SSAMB </h4>
+                     <hr>
+                     <p>In my opinion, women are tech are a big asset to the selves and the community. So I will encourage every girl to get into the tech industry and explore options.</p>
+                     </div>
+                 </div>
+                 <div>
+                     <!-- <button id="dropbtn">MORE <img class="dropimg" src="images/dropdown.png"></button> -->
+                    </div>
+                    <div class="hide"></div>
+                 <div class="speakers">
+                     <img class="images" src="images/Benimg.jpeg" alt="speakers imge">
+                     <div class="column">
+                     <h5>Steph Bahamas</h5>
+                     <h4>Co-founder of the SSAMB </h4>
+                     <hr>
+                     <p>Tech is the leading industry in the world. It comes with magically powers and well cherished when women invade the tech space.</p>
+                     </div>
+                 </div>
+                 <div class="speakers">
+                     <img class="images" src="images/Benimg.jpeg" alt="speakers imge">
+                     <div class="column">
+                     <h5>Steph Bahamas</h5>
+                     <h4>Co-founder of the SSAMB </h4>
+                     <hr>
+                     <p>The sole idea of advocating for black girls in tech is to bring value to every black girl and let them see the potential in them.</p>
+                     </div>
+                 </div>
+                 <div class="speakers">
+                    <img class="images" src="images/Benimg.jpeg" alt="speakers imge">
+                    <div class="column">
+                    <h5>Jane Cameroon</h5>
+                    <h4>Guest</h4>
+                    <hr>
+                    <p>The version, the goal, the technologies, the growth, every girl child should be a part of it. Lets make tech a comfort place for every child</p>
+                    </div>
+                </div>
```

Comment:
- [ ] The speakers cards are well structured 💪🏻 , they only need a margin bottom to separate them from each other

Reply:
Resolved.
Margin bottom added to separate speakers cards from each other.